odoo/models.py

```diff
@@ -2045,29 +2060,27 @@ def _read_group_orderby(self, order, groupby, query):
             order_match = regex_order.match(order_part)
             if not order_match:
                 raise ValueError(f"Invalid order {order!r} for _read_group()")
-            order_field = order_match['field']
-            if order_match['property']:
-                raise ValueError(f"Invalid order {order!r} for _read_group(): properties are not supported")
+            order_term = order_match['term']
             order_direction = (order_match['direction'] or 'ASC').upper()
             order_nulls = (order_match['nulls'] or '').upper()
 
-            if order_field not in groupby:
+            if order_term not in groupby_terms:
```

Comment:
groupby_terms keys are the same as previous groupby items, isn't it ? Can't we keep the naming order_field ? order_fname ? Actually "term" is quite vague to me.